source/languages/en/riak/community/index.md

```diff
@@ -11,116 +11,79 @@ moved: {
 }
 ---
 
-There is a vibrant and fast-growing community forming around Riak.
-Whether you need help or simply want to get involved, we are easy to
-contact.
+There is a vibrant, passionate and growing community forming around Riak. Whether you need help or simply want to get involved, we are easy to contact.
 
-## Help
+## Platforms
+We're big on answering questions for beginner and veteran users alike. To make sure we can answer them, we curate a number of discussions on a daily basis. 
```

Comment:
Since the last sentence is technically introducing a list, it'd be awesome if it ended with : , since : visually signals "I will provide this information below."